src/frontend/src/app/app.routes.ts

```diff
@@ -5,7 +5,7 @@ export const ROUTES: Routes = [
   {
     path: '',
     pathMatch: 'full',
-    redirectTo: 'open',
+    redirectTo: 'proposal',
```

Comment:
Maybe we can put the proposal list on this URL instead. We'll probably have `/proposals` as the base URL of the whole Angular app, so if we put `/proposal` on top of that it'll be `/proposals/proposal`.